helm/alfresco-content-services/tests/repository_test.yaml

```diff
@@ -0,0 +1,30 @@
+---
+suite: test repository manifest
+templates:
+  - deployment-repository.yaml
+  - config-repository.yaml
+  - secret-database.yaml
+  - secret-s3.yaml
+  - config-dev-log4j-properties.yaml
+tests:
+  - it: should have basic metadata in place
+    values: &testvalues
+      - values/test_values.yaml
+    asserts:
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-alfresco-cs-repository
+      - equal:
+          path: spec.template.metadata.annotations.checksum/config
+          value: 65b89491b06b9ccd8146424a8eab9d7a9d109a18db722e1276f1223ee37fd47c
+  - it: should have basic metadata in place
+    values: *testvalues
+    asserts:
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-alfresco-cs-repository-configmap
+        template: config-repository.yaml
+      - matchRegex:
+          path: data.CATALINA_OPTS
+          pattern: -Dencryption.ssl.truststore.location=/.*
```

Comment:
Can you also handle spaces and avoid using wildcards (we have a static string). See `search_test.yaml`.
Maybe also add an test where custom metadata keystore is used: https://github.com/Alfresco/acs-deployment/blob/ecbe4090e450cc7477a634e53f419b11987d3fbf/docs/helm/examples/custom-metadata-keystore.md

Reply:
rewritten it.

don't want to test custom metadata keystore given that logic is really simple (values plus a default) and that is out of scope for this pr